### DISCUSSION TITLE
Why does my state estimation yaw always approaches zero?

### DISCUSSION THREAD

**Discussion by SharyatSB** [bitcraze/discussions#927] [Q&A]
Hey all, 

I am using cfclient to test the yaw of the drone. Whenever I turn the drone in either direction, the yaw in the state estimator window shows a value, however if I keep the drone stationary after moving it the value keeps decreasing or increasing and always approaches 0. Could someone please explain why this happens and if something is wrong?

If the questions is vague and hard to understand, I can try recording my screen and sharing it. 

Thanks!

**Comment by knmcguire** [comment]
Hi! It would be better to record your plot and see if we can see the same thing.

I don't think that recording your screen is necessary. Make sure that you select enough samples, and if you do right click on the plotter, you'll see an option of 'export' for your plot.
